### ISSUE TITLE
Soft reset did not revert the "Space Colonist" achievement

### ISSUE THREAD

**Issue by ambre-m** [pmotschmann/Evolve#1140]
I was playing for the "Gross" achievement: have a greatness event with a non-custom race that fanatized "ooze".

I played ooze challenge to black hole, boomed it to magic universe, and wrongly played my custom race (at 4 stars) up to bioseed.
On reset, I obviously did not gained Gross, so I "restore backup" (to go back to the run) and "soft reset" (to go back at race selection).

Now, I chose magic universe with Pinguicula, and looked at achievements: "Space Colonist" is still accounted for in Magic universe, even if I never played magic before.
Other achievements where removed correctly (had 5 others.

TLDR: Magic Space Colonist was not removed after bioseed > restore backup > soft reset.

Those are my two exported saves
[evolve-2024-05-08-08-17.txt](https://github.com/user-attachments/files/15543207/evolve-2024-05-08-08-17.txt)
[evolve-2024-05-08-08-39.txt](https://github.com/user-attachments/files/15543206/evolve-2024-05-08-08-39.txt)
